resources/views/themes/default1/front/paynow.blade.php

```diff
@@ -121,78 +121,56 @@
 
 
                     </table>
-                    <hr class="tall">
-                    <!-- <h4 class="heading-primary">Cart Totals</h4> -->
-                   <!--  <div class="col-md-12">
-                        <table class="cart-totals">
-                            <tbody>
-
-
-                                <tr class="total">
-                                    <th>
-                                        <strong>Order Total</strong>
-                                    </th>
-                                    <td>
-                                        <strong><span class="amount"><small>{!! $symbol !!} </small> {{$invoice->grand_total}}</span></strong>
-                                    </td>
-                                </tr>
-
-                            </tbody>
-                        </table>
-                        <hr class="tall">
-                    </div> -->
 
                 </div>
-                {!! Form::open(['url'=>'checkout','method'=>'post']) !!}
+                {!! Form::open(['url'=>'checkout','method'=>'post','id' => 'checkoutsubmitform']) !!}
                   @if($invoice->grand_total > 0)
                 <h4 class="heading-primary">Payment</h4>
                     <?php $gateways = \App\Http\Controllers\Common\SettingsController::checkPaymentGateway($invoice->currency);
                       $rzpstatus = \App\Model\Common\StatusSetting::first()->value('rzp_status');
                        ?>
-                     @if($gateways) 
-                  <div class="form-group">
-
-                    <div class="col-md-6">
-                        @if(count($gateways)>0 ) 
-                  <div class="form-group">
+                    
+                
+                @if(count($gateways)) 
+                  <div class="row">
 
                     <div class="col-md-6">
                         @foreach($gateways as $gateway)
-                        {{ucfirst($gateway)}} {!! Form::radio('payment_gateway',strtolower($gateway)) !!}<br><br>
+                        <?php
+                          $processingFee = \DB::table(strtolower($gateway))->where('currencies',$invoice->currency)->value('processing_fee');
+                        ?>
+                        {!! Form::radio('payment_gateway',$gateway,false,['id'=>'allow_gateway']) !!}
+                         <img alt="Porto" width="111"  data-sticky-width="52" data-sticky-height="10" data-sticky-top="10" src="{{asset('client/images/'.$gateway.'.png')}}">
+                          <br><br>
+                         <div id="fee" style="display:none"><p>An extra processing fee of <b>{{$processingFee}}%</b> will be charged on your Order Total during the time of payment</p></div>
```

Comment:
why is this hidden?

Reply:
We only show this div when Stripe payment radio button is checked